package.json

```diff
@@ -87,7 +91,9 @@
     "bluebird": "^3.5.0",
     "body-parser": "^1.12.4",
     "bull": "^3.4.2",
-    "commander": "^2.13.0",
+    "chalk": "^2.4.1",
+    "cli-table": "^0.3.1",
+    "commander": "https://framagit.org/rigelk/commander.js.git#5b9cc4c6feb4f372f42d796cce76fc8c002174c8",
```

Comment:
:/

Reply:
It's either that or we don't have colors :/

Reply:
If they don't want the patch upstream, well no colors

Reply:
I'm not using the patch upstream, I'm developing it anew